modules/rosa-sts-creating-cluster.adoc

```diff
@@ -105,16 +106,24 @@ OIDC Endpoint URL:          https://rh-oidc.s3.us-east-1.amazonaws.com/<cluster_
 ====
 If installation fails or the `State` field does not change to `ready` after 40 minutes, check the installation troubleshooting documentation for more details.
 ====
-. Enter the following command to follow the OpenShift installer logs to track the progress of your cluster creation:
+. Set the environment variable for the `cluster_id` from the previous output:
 +
 [source,terminal]
 ----
-$ rosa logs install --cluster=<cluster_name> --watch
+$ export cluster_id=<cluster_id>
 ----
-+
+
+[IMPORTANT]
+====
+You must wait for the `State` output from `rosa describe cluster` to progress to
+"pending (Waiting for OIDC configuration)" before proceeding.
```

Comment:
We don;t typically use quotes, so I believe this should be with back ticks instead:  `pending (Waiting for OIDC configuration)`

Reply:
Does that message appear in the terminal like that? If so, JQ is right we need to remove the quotes.

Reply:
```suggestion
`pending (Waiting for OIDC configuration)` before proceeding.
```

Reply:
@sfortner-RH I took the exact output from the earlier `rosa describe cluster` example output. I've already addressed this with backticks 👍